src/cow-react/modules/limitOrders/pure/RateImpactIndicator/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { AlertTriangle } from 'react-feather'
+import styled from 'styled-components/macro'
+import { MouseoverTooltipContent } from 'components/Tooltip'
+import { LOW_RATE_THRESHOLD_PERCENT } from '@cow/modules/limitOrders/const/trade'
+
+interface RateImpactProps {
+  rateImpact: number
+}
+
+const PercentBox = styled.span<{ isPositive: boolean; isTooLowRate: boolean }>`
+  font-size: 12px;
+  display: inline-flex;
+  align-items: center;
+  margin-left: 5px;
+  color: ${({ isPositive, isTooLowRate }) => (isPositive ? 'green' : isTooLowRate ? 'red' : 'grey')};
+
+  svg {
+    margin-left: 5px;
+  }
+`
+const ImpactTooltip = styled.span`
+  padding: 5px 10px;
+`
+
+export function RateImpactIndicator({ rateImpact }: RateImpactProps) {
+  const isPositive = rateImpact > 0
+  const hugePositiveImpact = rateImpact > 999
+  const isTooLowRate = rateImpact < LOW_RATE_THRESHOLD_PERCENT
+
+  if (!rateImpact) return null
+
+  const tooltipContent = <ImpactTooltip>TODO: explain rate impact</ImpactTooltip>
```

Comment:
okok, let's not forget about this later. How do we plan to follow up these? 

![image](https://user-images.githubusercontent.com/2352112/200383447-c10f0ca0-6afd-4d4f-9f95-a3171ceaf472.png)

Reply:
When the main part is ready we should search a string `TODO` in the whole project and fix all of them (if it make sense of course)